compel/arch/aarch64/src/lib/infect.c

```diff
@@ -81,6 +99,8 @@ int compel_get_task_regs(pid_t pid, user_regs_struct_t *regs, user_fpregs_struct
 		goto err;
 	}
 
+	memset(&fpsimd->gcs, 0, sizeof(fpsimd->gcs));
```

Comment:
we don't memset(0) other register structs, why there is a need to clear fpsimd->gcs?